transform/mattermost-analytics/models/intermediate/product/nps/int_nps_feedback.sql

```diff
@@ -0,0 +1,41 @@
+-- Temporarily materialize during development
+{{
+    config({
+        "materialized": "table",
+        "snowflake_warehouse": "transform_l"
+    })
+}}
+
+with mattermost_nps as (
+select 
+    server_id as server_id
+    , user_id as user_id
+    , license_id as license_id
+    , event_date as event_date
+    , timestamp as timestamp
+    , server_version as server_version
+    , feedback as feedback
+    , user_role as user_role
+    , received_at as feedback_received_at
+    from {{ ref('stg_mattermost_nps__nps_feedback') }} 
+    where feedback is not null
+    qualify row_number() over (partition by user_id, feedback order by timestamp desc) = 1
```

Comment:
Is this the intended partition columns? It will keep any unique feedback per user_id.

Reply:
This is the intended partition, every feedback is captured.

Reply:
Then if the intention is to keep all feedbacks, is the qualify statement needed?

Reply:
the qualify statement is for de-duplication, instead of using distinct